src/aequitas/preprocessing.py

```diff
@@ -1,66 +1,62 @@
 import logging
-from sys import exit
-
 import pandas as pd
+from typing import List, Tuple
```

Comment:
FYI these are now deprecated – you can just use `list[str]`, _etc_.

That said, I don't know that this is permitted on 3.6, so this form makes sense for now.